model_security_adjust_oaw/models/supplier_stock.py

```diff
@@ -123,3 +127,18 @@ def create(self, vals):
             for action in server_actions:
                 action.sudo()._process(action, [res.id])
         return res
+    @api.multi
+    def _get_ordered_amount(self):
+        for ss in self:
+            sol= self.env['sale.order.line'].sudo().search([
+                    ('is_mto', '=', True),
+                    ('state', 'in', ('draft', 'sent')),
+                    ('product_id', '=', ss.product_id.id)
+                ])
+            print sol
+            i = 0
+            for line in sol:
+                if line.order_id.supplier_id == self.env.user.partner_id:
+                    print "Random"
+                    i += 1
```

Comment:
I think we have discussed before and we should use the quantity in the order line.